CHANGES.md

```diff
@@ -15,7 +15,20 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Delete unused Dockerfiles, fixes #349 and #352
```

Comment:
PR numbers and commit hash should use the full URL for generated docs (here and below)